ui/pages/confirmations/confirmation/components/existing-requests-banner-alert/existing-requests-banner-alert.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../../../shared/constants/metametrics';
+import {
+  BannerAlert,
+  BannerAlertSeverity,
+} from '../../../../../components/component-library';
+import { MetaMetricsContext } from '../../../../../contexts/metametrics';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import {
+  getQueuedRequestCount,
+  pendingConfirmationsSortedSelector,
+} from '../../../../../selectors';
+
+export const ExistingRequestsBannerAlert = () => {
+  const t = useI18nContext();
+  const trackEvent = useContext(MetaMetricsContext);
+
+  const [metricsSent, setMetricsSent] = useState(false);
+
+  const queuedRequestCount = useSelector(getQueuedRequestCount);
```

Comment:
If a user has multiple confirmations in the popup, with the first one being a "switch Ethereum chain" request for example, it would still be useful to know that other confirmations are in the queue (not in the popup). I confirmed we can keep this for all request types with Barbara.